nightly-playground/lib/common-tools-stack.ts

```diff
@@ -7,25 +7,60 @@ compatible open source license. */
 
 import { Stack, StackProps } from 'aws-cdk-lib';
 import { Certificate, CertificateValidation } from 'aws-cdk-lib/aws-certificatemanager';
+import {
+  Effect, ManagedPolicy, PolicyStatement, Role, ServicePrincipal,
+} from 'aws-cdk-lib/aws-iam';
 import { HostedZone } from 'aws-cdk-lib/aws-route53';
+import { Bucket } from 'aws-cdk-lib/aws-s3';
 import { Construct } from 'constructs';
 
 export class CommonToolsStack extends Stack {
-    readonly certificateArn: string
+  readonly certificateArn: string
 
-    readonly zone = 'playground.nightly.opensearch.org'
+  public readonly customRole: Role
 
-    constructor(scope: Construct, id: string, props: StackProps) {
-      super(scope, id, props);
+  readonly zone = 'playground.nightly.opensearch.org'
 
-      const route53HostedZone = new HostedZone(this, 'nigghhtlyHostedZone', {
-        zoneName: this.zone,
-      });
+  constructor(scope: Construct, id: string, props: StackProps) {
+    super(scope, id, props);
 
-      const certificate = new Certificate(this, 'cert', {
-        domainName: this.zone,
-        validation: CertificateValidation.fromDns(route53HostedZone),
-      });
-      this.certificateArn = certificate.certificateArn;
-    }
+    const route53HostedZone = new HostedZone(this, 'nigghhtlyHostedZone', {
```

Comment:
Typo?

Reply:
Oh! YEah cannot change that now. It will try to create same hosted zone again. Ignore for now.

Reply:
Sure.